blaze/expr/tests/test_arithmetic.py

```diff
@@ -64,8 +64,8 @@ def test_dir():
     i = symbol('i', '10 * int')
     d = symbol('d', '10 * datetime')
 
-    assert i + 1
-    with pytest.raises(Exception):
+    i + 1  # this works
```

Comment:
`assert` checks the non-zero-ness of the expressions which now triggers a call to `compute`